contrib/scripts/no-newlines-in-log-call.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Can we name this something more consistent like `check-log-newlines.sh`?

Reply:
Fixed.